Makefile

```diff
@@ -105,25 +105,35 @@ endif
 clean:
 	rm -rf dist *.tar vendor docker .go-pkg-cache
 
+PLUGIN_LOCATION?=$(CURDIR)/dist/libnetwork-plugin
```

Comment:
I'd rather see all of these adjustable constants defined at the top of each Makefile with a short comment for each.   But perhaps an overkill.

It feels like there is a general lack of commenting in this Makefile.  I realize that you can just read the code and work it out, but a short description of the make targets wouldn't go amiss.